### ISSUE TITLE
Autoupgrade - Upgrade to V9 - When you enable/disable any option , you'll have a red notification

### ISSUE THREAD

**Issue by AureRita** [PrestaShop/PrestaShop#36771]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

When I want to enable/disable any option, and I save, I have a bad request as you can see : 

[recording(274).webm](https://github.com/user-attachments/assets/170c451c-3cad-4543-a110-30aa68721402)

You can also reproduce from any version that upgrade to 9.0.0

### Expected behavior

I expected that I don't have any bad messages 

### Steps to reproduce

1. Install a fresh 8.0.5 
2. Install Autoupgrade (from dev branch because without it, you can't upgrade to v9) 
3. Upgrade from v8.0.5 to 9.0.0 thank to the local archive
4. enable "disable all overides" and clic on save 
5. :x: error 

### PrestaShop version(s) where the bug happened

9.0.0

### PHP version(s) where the bug happened

8.1

### If your bug is related to a module, specify its name and its version

autoupgrade v6.1.0 from dev branch 

### Your company or customer's name goes here (if applicable).

PS

**Comment by Quetzacoalt91** [comment]
I'm wondering if this is related to the URL rewriting. Can yo check your Apache has mod_rewrite enabled?

**Comment by M0rgan01** [comment]
@AureRita Did you see the comment from @Quetzacoalt91 ? could you retest ?

**Comment by AureRita** [comment]
Hi @Quetzacoalt91 and @M0rgan01, 

sorry for the time spend until I answer you, I tested it again with the version 6.3.0 and I continue to reproduce the issue as you can see : 

https://github.com/user-attachments/assets/fb6cf455-0c2a-469a-bd98-e082ef7a139c

the mod_rewrite seems to be enabled, If i'm right ? 

waiting for feedback

**Comment by M0rgan01** [comment]
@AureRita  With a "clean" starting shop I do not reproduce the problem. On the starting store, did you have a .htaccess at the root of the store? this one may have rules that lead you to this problem

https://github.com/user-attachments/assets/3fd752d3-fe3d-4b03-ba6e-b2c056d2d89c

**Comment by AureRita** [comment]
Hi @M0rgan01 

I had a .htaccess but even if I delete it, I still have the same issue as you can see : 

https://github.com/user-attachments/assets/b99c723c-4367-4044-96a9-72dffd2bec6b

**Comment by gericfo** [comment]
@AureRita can you test it again on the new UI?

**Comment by AureRita** [comment]
Hi ! 

I test it again with the new UI and currently, we have another issue on 9.0.0 before we can select an option as you can see : 

https://github.com/user-attachments/assets/b7936388-3db0-45ac-90db-62006e0e108b

I think it's because we can't check the difference between our current version and the other version. 

I tested to force the update option but it doesn"t want to go on the other step, maybe because it doesn't know where we can upgrade as you can see : 

https://github.com/user-attachments/assets/bdec6133-1442-4d51-8af5-3f0229adf3f6

so I can see if it's "corrected"

**Comment by AureRita** [comment]
I'll add more information about this two error : 

- about the check requirement, we have an error 500 when it does it infinite loading as you can see : 

![Image](https://github.com/user-attachments/assets/a5568544-79cc-4335-b896-2837034a4fa9)

- about the checking of an option in the option tab : 

![Image](https://github.com/user-attachments/assets/099aeb42-fb81-47a4-af91-f2064d997bd7)

we have this error on any option that we can check

**Comment by M0rgan01** [comment]
@AureRita The initial error is no longer the one you shared later, I have the impression that your ticket talks about different errors on different versions.

The latest screenshot is an error already reported here https://github.com/PrestaShop/PrestaShop/issues/38315

Can you confirm that the initial error is no longer present ?

**Comment by Quetzacoalt91** [comment]
Some of the errors are fixed with the beta.2.
Let's go back to the initial issue, with the double index.php found in the URL. Please confirm you still have it on the UI provided with the v7 version, the rest will go in other issues as the one reported by Morgan.

**Comment by AureRita** [comment]
Hi @Quetzacoalt91 

Even with the beta 2, I still have some error 500, it could be by this issue https://github.com/PrestaShop/PrestaShop/issues/38315 because it change the correctly the option when we return on the page but it seems I have other issue on it, I share you the video I made with other issue : 

https://github.com/user-attachments/assets/1eb7c05f-0aba-47d9-9a37-0529514b3337

I expect that can help you

**Comment by AureRita** [comment]
Seen with you, currently the issue showed at the end is not the same that the first one. Moreover, it seems to be an issue that is related to https://github.com/PrestaShop/PrestaShop/issues/38315 so I close this issue
